docs/essentials/test-summary.md

```diff
@@ -1,110 +1,129 @@
 ---
-description: This guide shows you how to optimize your Semaphore 2.0 workflow for monorepo projects.
+description: This document
 ---
 
-# Test Summary
+# Tests
 
-!!! beta "Feature in beta"
-    Beta features are subject to change.
+Semaphore collects XML test reports and uses them to provide insight into your
+pipelines.
 
-Working with tests in your projects should be efficient and easy. The bigger your project
-becomes, the harder it is to track all the failures in your workflows. Firstly
-you have to navigate to specific job and then search through the logs for failure messages:
+With test reports, you enable your team to get an effective and consistent view
+of your CI/CD test suite across different test frameworks. You get a clear
```

Comment:
Perhaps: consistent view of your test suite across different test frameworks and stages in a CI/CD workflow?